src/components/Page.vue

```diff
@@ -115,7 +115,8 @@ export default class Page extends Vue {
   }
 
   get languages() {
-    return Object.keys(Language).map(key => Language[key as keyof typeof Language]);
+    return Object.keys(Language).map(key => Language[key as keyof typeof Language])
+                      .sort((a: Language, b: Language) => (this.$t(a) > this.$t(b)) ? 1 : -1);
```

Comment:
Mind updating to use `localeCompare` instead? 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

Reply:
localCompare will NOT sort the languages alphabetically for translated values - it will only use the local settings for the characters interpretation